wlct/cogs/clot.py

```diff
@@ -636,30 +632,6 @@ async def linkd(self, ctx, arg="invalid_cmd", arg2="invalid_id"):
                         "Please enter a valid division id to link to this channel. Use ``bb!divisions`` to see list of divisions.")
                     return
                 for tournament in tournaments:
-                    # if a private tournament, the person sending this command must be linked and be the creator
-                    if tournament.private:
-                        player = await self.bot.bridge.getPlayers(discord_member__memberid=ctx.message.author.id)
-                        if len(player):
-                            player = player[0]
-                            if hasattr(tournament, 'parent_tournament'):
-                                if tournament.parent_tournament:
```

Comment:
Remind me why we're getting rid of this again?

Reply:
Trying to link tourneys right now is really restrictive in that you must be an admin of the discord server && be the creator of the tourney (unless it is CL which we hardcode for). This relaxes it to just being the admin. (Clan brawl was the main reason behind adding this since others couldn't link it)

Reply:
Got it, thanks!